### PR TITLE
nav bar position fixed, corrected some pages, Auth is added with sign up/log in

### PR DESCRIPTION
**Pull request by cgf5033**

### CODE REVIEW

**Review by Doobss — APPROVED** [review]

**Review by positivefx — APPROVED** [review]

File: client/src/components/Login.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+  return (
+    <>
+      <h1>Log in</h1>
+      <input
+        id="search"
+        type="search"
+        autoComplete="off"
+        maxLength="150"
+        style={{
```
Comment (MarkAlperin) [inline client/src/components/Login.js:35]: move inline styles to styled-component format for enhanced readability. I know it's work in progress :)

File: client/src/components/Nav.js [hunk 2]
```diff
@@ -0,0 +1,228 @@
+import React, { useState, useEffect } from 'react';
+import styled, {css} from 'styled-components';
+import Search from './Search';
```
Comment (MarkAlperin) [inline client/src/components/Nav.js:3]: I think (not sure about this) that the convention for imports is to do internal imports below external imports, with a blank line between

File: client/src/components/Nav.js [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+          width:'1.8em',
+          height:'auto',
+          color: 'black',
+          cursor: 'pointer'
+        }}
+        onClick={() => {
+          setMenu(!menu)
+        }}/>
+      </div>
+      {/* {loggedIn ? (
```
Comment (MarkAlperin) [inline client/src/components/Nav.js:94]: These comments are all work in progress stuff I assume

**Review by MarkAlperin — APPROVED** [review]
lgtm

**Pull request merged**